module/VuFindSearch/src/VuFindSearch/Service.php

```diff
@@ -356,11 +356,19 @@ function ($o) {
                 $args
             );
             if (!$response->stopped()) {
+                // We need to construct our error message differently depending
+                // on whether or not we have a command object...
+                $commandClass = isset($args['command'])
+                    ? get_class($args['command']) : 'null';
+                $context = $args['context'] ?? $commandClass;
```

Comment:
I might be missing something here, but why `$commandClass` instead of `$args['command']->getContext()`?

Reply:
Probably because when I discovered this problem, I was debugging an issue and hadn't fully refreshed my memory on how any of this works. :-) I'll correct it shortly; thanks for the correction!